Nucleo-F070RB cannot build by GCC_ARM since CMSIS5 merge

### Description
- Type: Bug
- Related issue: `#4459`, `#4294`
- Priority: Blocker

---------------------------------------------------------------
## Bug

**Target**
Nucleo-F070RB

**Toolchain:**
GCC_ARM

**Toolchain version:**
gcc-arm-none-eabi-5_2-2015q4-20151219-win32
gcc-arm-none-eabi-6-2017-q1-update

gcc-arm-none-eabi-5_4-2016q3 for linux

**Expected behavior**
Build finished and has bin file

**Actual behavior**
Build fail
```
../mbed-os/cmsis/core_cm0.h:591:12: error: #include expects "FILENAME" or <FILENAME>
   #include CMSIS_VECTAB_VIRTUAL_HEADER_FILE
            ^
make[1]: *** [code/ec_debug_uart.o] Error 1
make: *** [all] Error 2

```

**Steps to reproduce**
1. Export the code for GCC_ARM
2. Build by Toolchain

remove the CMSIS_VECTAB_VIRTUAL_HEADER_FILE on make and use "cmsis_nvic.h" to instate it can build pass.

--- a/tools/export/gnuarmeclipse/.cproject.tmpl
+++ b/tools/export/gnuarmeclipse/.cproject.tmpl
@@ -177,7 +177,7 @@
 								</option>
 								<option id="ilg.gnuarmeclipse.managedbuild.cross.option.assembler.defs.{{u.id}}" name="Defined symbols (-D)" superClass="ilg.gnuarmeclipse.managedbuild.cross.option.assembler.defs" useByScannerDiscovery="true" valueType="definedSymbols">
 									{% for s in opts['as']['defines'] %}
-									<listOptionValue builtIn="false" value="{{s}}"/>
+									<listOptionValue builtIn="false" value="{{s|replace("\"", "\\\"")|escape}}"/>
 									{% endfor %}
 								</option>
 								<option id="ilg.gnuarmeclipse.managedbuild.cross.option.assembler.include.files.{{u.id}}" superClass="ilg.gnuarmeclipse.managedbuild.cross.option.assembler.include.files" useByScannerDiscovery="true" valueType="includeFiles">
@@ -207,7 +207,7 @@
 								</option>
 								<option id="ilg.gnuarmeclipse.managedbuild.cross.option.c.compiler.defs.{{u.id}}" name="Defined symbols (-D)" superClass="ilg.gnuarmeclipse.managedbuild.cross.option.c.compiler.defs" useByScannerDiscovery="true" valueType="definedSymbols">
 									{% for s in opts['c']['defines'] %}
-									<listOptionValue builtIn="false" value="{{s}}"/>
+									<listOptionValue builtIn="false" value="{{s|replace("\"", "\\\"")|escape}}"/>
 									{% endfor %}
 								</option>
 								<option id="ilg.gnuarmeclipse.managedbuild.cross.option.c.compiler.include.files.{{u.id}}" name="Include files (-include)" superClass="ilg.gnuarmeclipse.managedbuild.cross.option.c.compiler.include.files" useByScannerDiscovery="true" valueType="includeFiles">
@@ -255,7 +255,7 @@
 								</option>
 								<option id="ilg.gnuarmeclipse.managedbuild.cross.option.cpp.compiler.defs.{{u.id}}" name="Defined symbols (-D)" superClass="ilg.gnuarmeclipse.managedbuild.cross.option.cpp.compiler.defs" useByScannerDiscovery="true" valueType="definedSymbols">
 									{% for s in opts['cpp']['defines'] %}
-									<listOptionValue builtIn="false" value="{{s}}"/>
+									<listOptionValue builtIn="false" value="{{s|replace("\"", "\\\"")|escape}}"/>
 									{% endfor %}
 								</option>
 								<option id="ilg.gnuarmeclipse.managedbuild.cross.option.cpp.compiler.include.files.{{u.id}}" name="Include files (-include)" superClass="ilg.gnuarmeclipse.managedbuild.cross.option.cpp.compiler.include.files" useByScannerDiscovery="true" valueType="includeFiles">

